### PR TITLE
Break tool parameter help words if necessary

### PR DESCRIPTION
**Pull request by guerler**
Avoids overflow scrolling and forces word breaks for tool parameter help details which were written without spaces between key words. See e.g. GATK tool in workflow editor. Without this fix, the tool form on the right can be scrolled horizontally within in the side panel. This PR also changes remaining word-break settings to word-wrap instead. The latter have wider browser support. Tested with FF, Safari and Chrome (see also: http://www.w3schools.com/cssref/css3_pr_word-wrap.asp). 

wordwrap

### CODE REVIEW

File: client/galaxy/style/less/ui.less [hunk 1]
```diff
@@ -152,7 +152,7 @@
             font-size: 0.8em;
             font-weight: normal;
             top: -1px;
-            word-break:break-word;
```
Comment (martenson) [inline client/galaxy/style/less/ui.less:155]: I think this had no effect before as `break-word` seems to not be a valid value (`word-break: normal|break-all|keep-all|initial|inherit;` http://www.w3schools.com/cssref/css3_pr_word-break.asp)

Comment (guerler) [inline client/galaxy/style/less/ui.less:155]: It did but not in FF. Its a webkit only feature: http://caniuse.com/#feat=word-break, so should not be used eitherway. Thanks for looking into it.

Comment (martenson) [inline client/galaxy/style/less/ui.less:155]: damn! thanks for the info!

Comment (martenson) [inline client/galaxy/style/less/ui.less:155]: I am revisiting this for purposes of https://github.com/galaxyproject/galaxy/issues/2591

Should we have a `break-word` on button titles?

**Pull request merged**